frontend/packages/console-dynamic-plugin-sdk/src/api/dynamic-core-api.ts

```diff
@@ -128,3 +121,19 @@ export const InventoryItemLoading: React.FC = require('@console/shared/src/compo
   .InventoryItemLoading;
 
 export { useFlag } from '../utils/flags';
+
+/**
+ * A lazy component that creates a monaco editor
+ * @example
+ * ```tsx
+ * <DroppableEditYAML
+ *     obj={obj}
+ *     create
+ *     kind={kindObj.kind}
+ *     resourceObjPath={resourceObjPath}
```

Comment:
Some of these props are not props we'd want to expose in the API. `kind` is not unique (and we don't want to expose the group~version~kind string). The caller shouldn't need to know `resourceObjPath`.

We might need to expose a subset of the internal component props.